src/components/common/Actions.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <v-container id="action-buttons-container" class="list-item">
+  <div id="action-buttons-container" class="list-item pa-0">
```

Comment:
Component should be smallest container possible -- parent can set whitespace according to its needs.

Reply:
PS "v-container" is the block in the middle of our apps that doesn't get wider after a certain max screen width.

Here, v-container was moved to the parent.